### PR TITLE
Feature/version replicator modernization

### PR DESCRIPTION
**Pull request by asha-pathik**
Modernization of version replicator  issue #2056

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/version-replicator/clientlibs/js/app.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+				    $scope.app.results = true;
+					$scope.results = data.result;
+					$('#results').show();
+					$scope.buildAgentInfoHTML($('#versionReplicator').find('input[name="cmbAgent"]:checked'));
+				}).error(function (data, status, headers, config) {
+				    $scope.app.running = false;
+				    $scope.app.results = false;
+                    $scope.app.error = true;
+                    $scope.showErrorMsg(data);
+                });
```
Comment (joerghoh) [inline ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/version-replicator/clientlibs/js/app.js]: please fix indentation

File: ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/version-replicator/includes/status.jsp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <div id="replication-agents-info">
+            <h2>Your replication request has been submitted.</h2>
+            <p>This request may take some time to complete. Please review the Replication agent logs and
+                configurations the complete replication status does not appear below shortly.</p>
+            <div class="message"></div>
+        </div>
+        <div id="replication-queue-message"></div>
+    </div>
+
+</section>
```
Comment (joerghoh) [inline ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/version-replicator/includes/status.jsp]: Does this need to be a JSP? It contains only HTML markup.

Comment (asha-pathik) [reply ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/version-replicator/includes/status.jsp]: Fixed. Please check

File: ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/version-replicator/clientlibs/js/app.js (hunk 1 above)
Comment (asha-pathik) [reply ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/version-replicator/clientlibs/js/app.js]: Fixed. Please check

**Comment by badvision** [comment]
@joerghoh Are the changes satisfactory?

**Comment by joerghoh** [comment]
LGTM.

**Pull request merged**